test/gas/debt_bridge/full/from_maker/2_MAKER-AAVE.mock.test.js

```diff
@@ -0,0 +1,329 @@
+const { expect } = require("chai");
+const hre = require("hardhat");
+const { deployments } = hre;
+const GelatoCoreLib = require("@gelatonetwork/core");
+
+const setupMAKERAAVE = require("./helpers/setupMAKER-AAVE.mock");
+const mockExecMAKERAAVE = require("./helpers/services/exec-MAKER-AAVE.mock");
```

Comment:
Let' stick to `camelCase` :

```suggestion
const setupMakerAave = require("./helpers/setupMakerAave.mock");
const mockExecMakerAave = require("./helpers/services/execMakerAave.mock");
```